tools/downloader/requirements.in

```diff
@@ -1,2 +1,3 @@
 pyyaml
 requests
+test-generator
```

Comment:
Why did you add this package here? In fact, I couldn't find its usage in your code.

Reply:
It was a missing req when running the converter. Hoeever, you are right, it should be part of the dldt req. Removed.